src/logic/nfts/collections.ts

```diff
@@ -235,6 +235,18 @@ export function getCollectionsExtraWhere(options: NFTFilters) {
   if (options.itemId) {
     extraWhere.push('itemBlockchainId: $itemId')
   }
+  if (options.creator) {
+    const creators = Array.isArray(options.creator)
+      ? options.creator
+      : [options.creator]
+    if (creators.length > 0) {
+      extraWhere.push(
+        `item_: { 
```

Comment:
is the last underscore intended?

Reply:
yeah, it's for nesting inside the `item` derived entity